cypress/integration/channel/add_users_to_channel_spec.js

```diff
@@ -0,0 +1,136 @@
+
+                // # Select 'Channel test 1' channel on the left hand side
+                cy.get(`#sidebarItem_${channelTest1Id}`).click();
+
+                // * 'Channel test 1' shoul be visible at the top of the center pane
+                cy.get('#channelHeaderTitle').should('contain', 'Channel test 1');
+
+                // # Then click it to access the drop-down menu
+                cy.get('#channelHeaderTitle').click();
+
+                // * The dropdown menu of the channel header should be visible;
+                cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+                // # Click 'Add Members'
+                cy.get('#channelAddMembers').click();
+
+                // * Assert that modal appears
+                cy.get('#addUsersToChannelModal').should('be.visible');
+
+                // # Click the first row clickable of the modal to select a user
+                cy.get('#multiSelectList').first().click();
+
+                // # Click the button "Add" to add this user to 'Channel test 1' channel
+                cy.get('#saveItems').click();
+
+                // # Wait for the modal to disappear
+                cy.get('#addUsersToChannelModal').should('not.exist');
+
+                cy.getLastPostId().then((id) => {
+                    // * The system message should contain 'added to the channel by you'
+                    cy.get(`#postMessageText_${id}`).should('contain', 'added to the channel by you');
+
+                    // # Verify username link
+                    verifyMentionedUserAndProfilePopover(id);
+                });
+            });
+        });
+    });
+
+    it('Combined Users: Usernames are links, open profile popovers', () => {
+        cy.getCurrentTeamId().then((teamId) => {
+            // # Create 'Channel test 2'
+            cy.apiCreateChannel(teamId, 'channel-test-2', channelDisplayNameTest2).then((res) => {
+                channelTest2Id = res.body.name;
+
+                // # Select 'Channel test 2' channel on the left hand side
+                cy.get(`#sidebarItem_${channelTest2Id}`).click();
+
+                // * 'Channel test 2' should be visible at the top of the center pane
+                cy.get('#channelHeaderTitle').should('contain', 'Channel test 2');
+
+                // # Then click it to access the drop-down menu
+                cy.get('#channelHeaderTitle').click();
+
+                // * The dropdown menu of the channel header should be visible;
+                cy.get('#channelHeaderDropdownMenu').should('be.visible');
+
+                // # Click 'Add Members'
+                cy.get('#channelAddMembers').click();
+
+                // * Assert that the modal appears
+                cy.get('#addUsersToChannelModal').should('be.visible');
+
+                // # Click 3 times the first row clickable of the modal to select 3 different users
+                cy.get('#multiSelectList').first().click();
```

Comment:
@VolatianaYuliana I think starting here is where the tests start to differ, but before here they share a lot of the same code. Can you rollup the shared "setup" stuff into a function? Maybe `addUsersToTeam(amount)`? And that function could click on `cy.get('#multiSelectList').first().click();` however many times you need it to?